layouts/footer.slim

```diff
@@ -5,5 +5,9 @@ footer.ui.center.aligned.container lang='en'
 script src="/static/dist/semantic-ui/semantic.min.js"
 script src="/static/javascripts/ga.js"
 script src="/static/javascripts/jquery.address.js"
+script src="/static/javascripts/cookie.js"
+script src="/static/javascripts/clipboard.js"
+script src="/static/javascripts/less.js"
```

Comment:
Not quite sure about this. I've tested by remove this line and the code clipboard don't work anymore.

So keep it here, I think it's OK right now.